lib/mayaUsd/ufe/UsdTransform3d.cpp

```diff
@@ -60,6 +62,26 @@ Ufe::Matrix4d primToUfeExclusiveXform(const UsdPrim& prim, const UsdTimeCode& ti
     Ufe::Matrix4d     xform = convertFromUsd(usdMatrix);
     return xform;
 }
+
+std::string isAttributeEditAllowed(const PXR_NS::UsdPrim& prim, const std::string& tokenName)
+{
+    std::string errMsg;
+
+    // check for xformOp:tokenName in XformOpOrderAttr first
+    UsdGeomXformable xformable(prim);
+    if (!MayaUsd::ufe::isAttributeEditAllowed(xformable.GetXformOpOrderAttr(), &errMsg)) {
+        MGlobal::displayError(errMsg.c_str());
+    } else {
+        // check for xformOp:tokenName
+        const TfToken xlate(tokenName);
+        if (!MayaUsd::ufe::isAttributeEditAllowed(prim.GetAttribute(xlate), &errMsg)) {
+            MGlobal::displayError(errMsg.c_str());
+        }
+    }
+
```

Comment:
This is a wrapper function around MayaUsd::ufe::isAttributeEditAllowed to avoid code duplication in UFE1.0 and UFE2.0 code base.